tools/capistrano/lib/inviqa_cap/notify_slack.rb

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+
+require 'capistrano'
+
+module InviqaCap
+  class NotifySlack
+    def self.load_into(config)
+      config.load do
+        set :domain, '{{hostname}}'
+        set :slack_channel, '#{{name}}'
+        set :github_url, 'https://github.com/{{git_url}}'.gsub!('git@github.com:', '')
+        set :jira_url, 'https://jira.example.com/'
+        set :jira_project, '\\(PROJECT1|PROJECT2|PROJECT3\\)' # regular expression!
+
+        set :slack_hook_url, 'https://hooks.slack.com/services/.../.../...'
```

Comment:
these project-specific settings are best outside the script e.g. maybe in deploy.rb